compiler/frontend/src/org/jetbrains/kotlin/resolve/extensions/SyntheticResolveExtension.kt

```diff
@@ -62,17 +63,19 @@ interface SyntheticResolveExtension {
                 // todo revert
                 override fun generateSyntheticMethods(
                     thisDescriptor: ClassDescriptor, name: Name,
+                    ctx: BindingContext,
```

Comment:
Would be nice to name this `bindingContext` or `context`, since we don't usually use abbreviations for identifier names in our project

Reply:
I will rename it.